fastai/transforms.py

```diff
@@ -61,6 +61,26 @@ def center_crop(im, min_sz=None):
     start_c = math.ceil((c-min_sz)/2)
     return crop(im, start_r, start_c, min_sz)
 
+def googlenet_resize(im, targ, min_area_frac, min_aspect_ratio, max_aspect_ratio, flip_p):
+    h,w,*_ = im.shape
+    area = h*w
+    for _ in range(10):
+        targetArea = random.uniform(min_area_frac, 1.0) * area
```

Comment:
Are the some assumptions here that the original image is big enough? What happens if it's really small? What happens if it's really big? Could you show a couple of examples - maybe link from the PR to a gist that gives this transform a really good work-out?

Reply:
I have used cifar-10 images for small size images, dogs vs cats for medium size images and downloaded few images of around 1920*1080 images from the web for large size images.
Link for gist: https://gist.github.com/supreethmanyam/11d4daadd799c6a1a3234745f65430b6